R/knitr-engine.R

```diff
@@ -625,7 +625,9 @@ eng_python_autoprint <- function(captured, options) {
 
   } else if (isHtml && py_has_method(value, "_repr_html_")) {
 
-    data <- as_r_value(value$`_repr_html_`())
+    py_capture_output({
```

Comment:
For some reason `_repr_html_` on the GH machine is also printing to stdout (with this we capture and discard the stdout output).